### ISSUE TITLE
WP table prefixes with both upper and lower case letters can cause DB query errors

### ISSUE THREAD

**Issue by MafuraG** [snicco/fortress-docs#1]
A WP site has the following table prefix

```
$table_prefix = 'mCp11111_';
```

The site database table all have a prefix respecting the case of the `$table_prefix`:
```
mCp11111_options
mCp11111_users
mCp11111_usermeta
....
```

When running the following command:
```
wp snicco/fortress password upgrade-legacy-hashes
```

The following DB error appears:
```
snicco_fortress.cli.ERROR Uncaught exception while running command [snicco/fortress password upgrade-legacy-hashes] 
    Snicco\ScopedVendor\Snicco\Component\BetterWPDB\Exception\QueryException "Table 'eQk_network_co_uk.mcp11111_users' doesn't exist
Query: [select `id`, `user_pass` from mcp1111_users where legacy_hash is null order by `id` asc limit ? for update]
Bindings: [501]" in /var/www/XXXXXX/htdocs/wp-content/mu-plugins/gridpane-snicco-fortress/snicco-fortress/vendor/snicco/better-wpdb/src/Exception/QueryException.php:39
```

The above is due to MySQL being case sensitive ie `eQk_network_co_uk.mcp11111_users` is not the same as `eQk_network_co_uk.mCp11111_users`

**Comment by calvinalkan** [comment]
We traced this to a bug in your [BetterWPDB](https://github.com/snicco/better-wpdb) library. 

https://github.com/snicco/snicco/issues/182

**Comment by calvinalkan** [comment]
Fixed in 1.0.0-beta.16
